.github/workflows/release.yml

```diff
@@ -1,29 +1,38 @@
-name: release
+name: 🦋 Changesets Release
+
 on:
   push:
     branches:
       - main
-concurrency: ${{ github.workflow }}-${{ github.ref }}
-env:
-  CI: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I don't know that we want to add this - since it can leave gaps in our publishes.  I.e., if `.pre.0` is in process and we follow it up quickly with a new change that starts a `.pre.1` release - we do want both workflows to run to completion.